apigw-dynamodb-python-cdk/example-pattern.json

```diff
@@ -0,0 +1,53 @@
+{
+  "title": "API Gateway direct integration to DynamoDB",
+  "description": "Create direct integration with API Gateway to DynamoDB showcasing transformation of request/response using VTL and CDK and implement examples for using Cognito, Lambda authorizer and API keys.",
```

Comment:
Description here needs to be lesser than or equal to 150

Reply:
Description was changed